plaidml/edsl/__init__.py

```diff
@@ -669,17 +669,42 @@ def call(fn, *args):
 
 
 def abs(x):
-    """A placeholder docstring."""
+    """Computes the elementwise absolute value of ``x``.
+
+        Args:
+            x (Tensor): The tensor used to peform the elementwise ``abs``.
+
+        Returns:
+            y (Tensor): The result of the elementwise ``abs`` operation.
```

Comment:
Are we required to name the output? I'd rather not if we don't have to

Reply:
Just checked - we are _not_ required to name outputs. I was just doing it because the template I was using had named the outputs.

I'll fix this in my next go-around. I didn't like the inconsistency of it either...